amp.py

```diff
@@ -1,15 +1,16 @@
 import os
 
+from PyQt5 import QtWidgets
+
 from importlib import import_module
 
 # tools for loading amp plugins
-# TODO: Reimplement to build plugin from a *.py file
 
 
-def load_plugin(ui_file, main_viewer):
+def load_plugin(ui_file: str, main_viewer: QtWidgets.QMainWindow) -> QtWidgets.QMainWindow:
     imp_name = os.path.basename(ui_file).split('.')[0]
     plugin = import_module(imp_name)
-    builder = getattr(plugin, 'buildAsPlugin')
+    builder = getattr(plugin, 'build_as_plugin')
```

Comment:
Snakes > camels 😆

Reply:
sadly, pyqt insists on CamelCase for literally everything, which means I have an egregious mix of snake_case and CamelCase in some places 🙄 😭